### PR TITLE
com.github.bluesabre.darkbar 1.0.0

### PR DESCRIPTION
**Pull request by bluesabre**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [ ] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
  - Reading .desktop and icon files needed to display application details
    - '--filesystem=host:ro'
    - '--filesystem=/var/lib/flatpak/app:ro'
    - '--filesystem=xdg-data/flatpak/app:ro'
  - Reading the FreeDesktop.org dark style user preference
    - '--system-talk-name=org.freedesktop.Accounts'
  - Reading/writing user preferences for Darkbar
    - '--talk-name=org.gnome.SettingsDaemon'
  - Creating an autostart file to launch Darkbar in the background at startup
    - '--filesystem=xdg-config/autostart:create'
  - Spawning host `xprop` (for setting window decorations) and `xdotool` (for processing XWayland windows) commands
    - '--talk-name=org.freedesktop.Flatpak'
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [ ] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*
  - N/A

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by hfiguiere** [comment]
bot, build com.github.bluesabre.darkbar

**Comment by hfiguiere** [comment]
```
builddir/files/share/appdata/com.github.bluesabre.darkbar.appdata.xml: FAILED:
• tag-missing           : <release> required
Validation of files failed
```

**Comment by bluesabre** [comment]
bot, build com.github.bluesabre.darkbar

File: com.github.bluesabre.darkbar.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+app-id: com.github.bluesabre.darkbar
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+command: com.github.bluesabre.darkbar
+finish-args:
+  - '--filesystem=host:ro'
+  - '--filesystem=xdg-config/autostart:create'
```
Comment (barthalion) [inline com.github.bluesabre.darkbar.yml:8]: It probably doesn't particularly matter, given how many permissions the app already has, but autostart should be ideally handled with the background portal: https://flatpak.github.io/xdg-desktop-portal/#gdbus-method-org-freedesktop-portal-Background.RequestBackground

**Comment by barthalion** [comment]
It's not immediately clear to me how to use it, as no windows show app in the Active Applications on my Fedora Silverblue 35.

**Comment by bluesabre** [comment]
> It's not immediately clear to me how to use it, as no windows show app in the Active Applications on my Fedora Silverblue 35.

Make sure your host has `xprop` installed, and if you're using Wayland, `xdotool`. Then GTK2 apps like GIMP or electron-based apps should show in the window list.

**Comment by bluesabre** [comment]
bot, build com.github.bluesabre.darkbar

**Pull request closed without merging**